presto/utils.py

```diff
@@ -366,11 +349,8 @@ def process_parquet(df: pd.DataFrame) -> pd.DataFrame:
         )
     df_pivot = df_pivot[~samples_with_too_few_ts]
```

Comment:
this can move inside the `if` for clarity

Reply:
done 74c9ae685b0250c564b4798136af9bb2eb95a108